docs/user/guides/canonical-urls.rst

```diff
@@ -0,0 +1,32 @@
+How to Enable Canonical URLs
+============================
+
+In this guide, we introduce relevant settings for enabling canonical URLs in popular documentation frameworks.
+
+.. seealso::
+
+   If you want to customize the domain from which your documentation project is served, please refer to :doc:`/guides/custom-domains`.
+
+Sphinx
+~~~~~~
+
+If you are using :doc:`Sphinx </intro/getting-started-with-sphinx>`,
+Read the Docs will automatically add a default value of the html_baseurl_ setting matching your canonical domain.
+
+If you have defined ``html_baseurl`` in your ``conf.py``, you need to ensure that the value is correct.
+
+.. _html_baseurl: https://www.sphinx-doc.org/page/usage/configuration.html#confval-html_baseurl
+
+MkDocs
+~~~~~~
+
+For :doc:`MkDocs </intro/getting-started-with-mkdocs>` we do not define your canonical domain automatically,
+but you can use the site_url_ setting to set a similar value.
```

Comment:
This also feels very limited. We should at least add another couple lines of explanation copy of what to set that as. Probably good to be explicit about what to set things to when using subdomains or custom domains on RTD.  An example code block is probably best.

Reply:
Agreed. Since we have an mkdocs example project, I can quickly go and see what can be done there :innocent:

Reply:
Haha, well that was a :rabbit: :hole:

Reply:
This part now has a copy-paste friendly code example. Unfortunately, there is a bit of limitations in how MkDocs is supported so it required IMO a reference to currently open issues in order to be fully explained.